Sources/Quick/QuickSpec.m

```diff
@@ -115,10 +115,23 @@ + (SEL)addInstanceMethodForExample:(Example *)example {
     }
 
     const char *types = [[NSString stringWithFormat:@"%s%s%s", @encode(id), @encode(id), @encode(SEL)] UTF8String];
-    NSString *selectorName = [NSString stringWithFormat:@"%@_%@_%ld",
-                              example.name.qck_selectorName,
-                              sanitizedFileName,
-                              (long)example.callsite.line];
+    
+    NSString *originalName = example.name.qck_selectorName;
+    NSString *selectorName = originalName;
+    unsigned int i = 2;
+    
+    static NSMutableSet<NSString *> *selectorNames;
```

Comment:
I'd like to make this more prominent, so it's obvious that we're maintaining global state. Is it possible to move this up to where `currentSpec` is defined, at the top of the file? Alternatively, we could move this into it's own singleton class--think `[QCKSelectorNameStore sharedStore]`, or something.